packages/foreman/foreman-proxy/foreman-proxy.spec

```diff
@@ -40,8 +40,8 @@ Requires:       rubygem(logging) < 3.0.0
 Requires:       rubygem(sinatra)
 
 # start specfile default Requires
-Requires: rubygem(concurrent-ruby) >= 1.0
-Requires: rubygem(concurrent-ruby) < 2.0
+Requires: rubygem(smart_proxy)
```

Comment:
Hmm, not sure where this came from but this is not intended. Looks like the script needs some tweaking :(